torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -76,12 +81,10 @@ bool isSimpleMap(Node *node) {
   // Make sure that the node doesn't broadcast.
   JIT_ASSERT(node->inputs().size() > 0);
   TensorType* expected_type = node->inputs()[0]->type()->cast<TensorType>();
-  if (!expected_type)
-    return false;
+  if (!expected_type) return false;
   static const auto equal_modulo_strides = [](TensorType* expected, const TypePtr& _actual) {
-    TensorType* actual = _actual->cast<TensorType>();
-    return actual &&
-           expected->scalarType() == actual->scalarType() &&
```

Comment:
scalarType test purposely dropped here, right? Would be a good to insert a cross-reference to the comment above, so that it's easy to remember why.